src/mozilla-traffic-cop.js

```diff
@@ -26,11 +26,19 @@ if (typeof Mozilla === 'undefined') {
  *
  *
  * @param Object config: Object literal containing the following:
- *      String id (required): Unique-ish string for cookie identification.
+ *      [String] id (required): Unique-ish string for cookie identification.
  *          Only needs to be unique to other currently running tests.
- *      Number cookieExpires (optional): Number of hours browser should remember
- *          the variation chosen for the user. Defaults to 24 (hours). A value
- *          of 0 will result in a session-length cookie.
+ *      [Number] cookieExpires (optional): Number of hours browser should
+ *          remember the variation chosen for the user. Defaults to 24 (hours).
+ *          A value of 0 will result in a session-length cookie.
+ *      [Boolean] storeReferrerCookie (optional): Flag to specify whether or not
+ *          original HTTP referrer should be placed in a cookie for later use.
+ *          Defaults to true.
+ *      [Function] customCallback (optional): Arbitrary function to run when
+ *          a variation (or lack thereof) is chosen. This function will be
+ *          passed the variation value (if chosen), or the value of
+ *          noVariationCookieValue if no variation was chosen. *Specifying this
```

Comment:
At first I was a little confused why there we're two separate experiments on page three, but after reading this comment and the code below I understand the callback takes precedence. Do we think this will be flexible enough for our needs, or could we potentially see ourselves doing experiments that could want both redirects and callbacks, while still ensuring segregation of variants? Would we ever want to do both a redirect and a callback in a single variation?

Having two separate experiments on page three works here, but it assumes the visitor is OK to see multiple experiments at once. Could this skew data?